index.ts

```diff
@@ -35,13 +35,21 @@ export enum HttpCodes {
     GatewayTimeout = 504,
 }
 
+/**
+ * Returns the proxy URL, depending upon the supplied url and proxy environment variables.
+ * @param serverUrl  The server URL where the request will be sent. For example, https://api.github.com
+ */
+export function getProxyUrl(serverUrl: string): string {
```

Comment:
Public function now exported. I'm planning to use this in checkout to add the server url to the git config